cli/command/image/remove.go

```diff
@@ -56,9 +57,13 @@ func runRemove(dockerCli command.Cli, opts removeOptions, images []string) error
 	}
 
 	var errs []string
+	var unresolvableErr = false
```

Comment:
minor: I'm not sure about the term "resolvable" here. Maybe "unrecoverable"  or just "fatal" ?